tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh

```diff
@@ -42,9 +60,11 @@ if [ -n "$CHANGED_FILES" ]; then
   files=$(comm -12 <(echo $files | tr ' ' '\n' | sort -u) <(echo $CHANGED_FILES | tr ' ' '\n' | sort -u))
 fi
 
+FILES_PER_PROCESS="$(expr $(echo "$files" | grep -o '\n' | wc -l) / $CPU_COUNT + 1)"
```

Comment:
I expect there's a sweet spot somewhere between `1` and `files/cpus` here -- there's likely to be one batch here that's far more expensive than the others and we spend undue time waiting for it to complete.

I'd suggest quickly trying `files/cpus/2`, `files/cpus/5`, `files/cpus/10` before submission and choosing the best (the additional divisor would create smaller batches so that we'd be able to more effectively backfill cpus that finish earlier).

Reply:
This might depend on the system specs, right? Number of cores and HD/SSD speed, for example. I suspect you're right, but I'll leave further environment tuning for another time.

Reply:
On my 128 core system with SSD:

files/cpus/1: 20s
files/cpus/2: 22s
files/cpus//3: 27s
files/cpus//5: 38s